docs/modules/rediscache.md

```diff
@@ -64,6 +73,15 @@ Updates existing Redis cache.
 | `skuCapacity`      | `integer` | The size of the Redis cache to deploy.  Valid values: for C (Basic/Standard) family (0, 1, 2, 3, 4, 5, 6), for P (Premium) family (1, 2, 3, 4). They denotes real size  (250MB, 1GB, 2.5 GB, 6 GB, 13 GB, 26 GB, 53GB) and (6 GB, 13 GB, 26 GB, 53GB) respectively. **Note**: you can only update from a smaller capacity to a larger capacity, the reverse is not allowed. | N        |
 | `enableNonSslPort` | `string`  | Specifies whether the non-ssl Redis server port (6379) is enabled. Valid values: (`enabled`, `disabled`) | N        |
 
+For `premium` plan, following updating parameter is available:
+
+| Parameter Name                                       | Type      | Description                                                  | Required                                                     |
+| ---------------------------------------------------- | --------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `redisConfiguration`                                 | `object`  | Redis Settings. See below possible keys.                     | N                                                            |
+| `redisConfiguration`.`rdb-backup-enabled`            | `string`  | Specifies whether RDB backup is enabled. Valid values: (`enabled`, `disabled`) | N                                                            |
+| `redisConfiguration`.`rdb-backup-frequency`          | `integer` | The frequency doing backup in minutes. Valid values: ( 15, 30, 60, 360, 720, 1440 ) | Yes when `rdb-backup-enabled` is set to `enabled`; otherwise is not required. |
```

Comment:
Is it "not required" or "invalid"?

Reply:
"invalid" is more precise, I'll change to "invalid", thanks.